packages/core/src/mixins/entity.js

```diff
@@ -34,7 +34,7 @@ module.exports = mixin(
                 this._app = parent.source._app;
                 this.name = utils.slugify(name.toLowerCase());
                 this.handle = this._handle(config);
-                this.label = config.label || this._label(config);
+                this.label = config.label || this._label(name);
```

Comment:
The [Doc class](https://github.com/frctl/fractal/blob/main/packages/fractal/src/api/docs/doc.js#L26) expects `_label` method be passed the config object, not name.